Add NuGet package configuration details for KaitaiStruct.Runtime.CSharp.

Arlorean opened the pull request:
Changes needed to create a NuGet package from the project for .NET 4.5+.
Package can be [downloaded from here](https://www.nuget.org/packages/KaitaiStruct.Runtime.CSharp/).

Arlorean committed:
Add NuGet package configuration details for KaitaiStruct.Runtime.CSharp.

GreyCat commented on kaitai_struct_runtime_csharp.csproj:
```diff
@@ -1,12 +1,12 @@
-    <RootNamespace>kaitai_struct_runtime_csharp</RootNamespace>
-    <AssemblyName>kaitai_struct_runtime_csharp</AssemblyName>
+    <RootNamespace>KaitaiStruct</RootNamespace>
+    <AssemblyName>KaitaiStruct.Runtime.CSharp</AssemblyName>
```
As far as I understand:

* We use just `Kaitai` namespace everywhere in .NET => probably "root namespace" should be just "Kaitai", although I'm not sure what it affects.
* Assembly name should be probably `Kaitai.Struct.Runtime` or something like that:
  * Judging from Microsoft's recommendations, first component of the assembly name should be organization, then the project, subproject, etc.
  * There would be probably no point to add `CSharp`, as this assembly would work equally well for any .NET language (C#, C++, F#, VB.NET, etc), and it is actually not that important what it is written in originally.

GreyCat commented:
Thanks for your contribution — it's been a long-awaited packaging effort for a nuget package. I've added minor note about naming — could you take a look?

Arlorean commented:
I’ll update the name to `Kaitai.Struct.Runtime` but I won’t publish a new package until there’s a Kaitai NuGet.org account.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

Arlorean committed:
Update assembly name to Kaitai.Struct.Runtime.
Update default namespace to Kaitai.
Generate XML documentation for project nuget.

Arlorean committed:
Correct spelling of Kaitai.

GreyCat commented:
Thanks, merging in!

Arlorean commented:
Here's a build version (with the correct spelling). Unzip it and upload it to the nuget.org account you setup. Or let me know the username/password and I can do it. 

[Kaitai.Struct.Runtime.0.7.0.nupkg.zip](https://github.com/kaitai-io/kaitai_struct_csharp_runtime/files/1652823/Kaitai.Struct.Runtime.0.7.0.nupkg.zip)
